src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sulu\Bundle\MediaBundle\Media\ImageConverter;
 
+use Imagine\Exception\NotSupportedException;
```

Comment:
Is it safe to use this also with versions of the imagine library not having this exception? If not we would have to rise the minimum version of it in `composer.json`. On the other hand I think it is not necessary to catch this exception separately, if we already catch the `RuntimeException` (hopefully soon the one from Imagine if the imagine-vips adapter releases a new version), which the `NotSupportedException` is inheriting from.

Reply:
have removed the `NotSupportedException ` from the catch clause